ta/crypt/include/derive_key_taf.h

```diff
@@ -0,0 +1,15 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2019, Linaro Limited
+ */
+
+#ifndef DERIVE_KEY_TAF_H
+#define DERIVE_KEY_TAF_H
+
+#include <pta_system.h>
```

Comment:
No need to include `<pta_system.h>` here, `<tee_internal_api.h>` would be more appropriate.

Reply:
Ah, sloppy/careless mistake from my side, I just copy pasted from `seed_rng_taf.h` where I initially had the code. Will fix.